var/spack/repos/builtin/packages/py-xdot/package.py

```diff
@@ -35,11 +36,12 @@ class PyXdot(PythonPackage):
     @run_after('install')
     def post_install(self):
         spec = self.spec
-        repo_paths = '%s:%s:%s:%s' % (
+        repo_paths = '%s:%s:%s:%s:%s' % (
             join_path(spec['pango'].prefix.lib, 'girepository-1.0'),
             join_path(spec['atk'].prefix.lib, 'girepository-1.0'),
             join_path(spec['gdk-pixbuf'].prefix.lib, 'girepository-1.0'),
-            join_path(spec['gtkplus'].prefix.lib, 'girepository-1.0'))
+            join_path(spec['gtkplus'].prefix.lib, 'girepository-1.0'),
+            join_path(spec['harfbuzz'].prefix.lib, 'girepository-1.0'))
```

Comment:
Should we be using `spec['foo'].libs.directories[0]` to properly handle cases where the libraries are in `lib64`, not `lib`?

Reply:
this is for .typelib files, not .so files, so I don't think they will ever be installed in lib64

Reply:
What is a `.typelib` file?

Reply:
I'm no expert on this, so I will just refer to https://gi.readthedocs.io/en/latest/

Reply:
Is this something that xdot creates and writes to these installation prefixes?

Reply:
xdot does not create them, they are created by the dependent packages (harfbuzz, pango, etc.)